apps/financial-aid/web-veita/src/utils/useMunicipalities.ts

```diff
@@ -0,0 +1,95 @@
+import { useEffect, useState } from 'react'
+
+import {
+  Municipality,
+  useAsyncLazyQuery,
+} from '@island.is/financial-aid/shared/lib'
+import { gql } from '@apollo/client'
+
+const MunicipalityQuery = gql`
+  query GetMunicipalityQuery {
+    municipalityByIds {
+      id
+      name
+      homepage
+      active
+      municipalityId
+      email
+      rulesHomepage
+      individualAid {
+        ownPlace
+        registeredRenting
+        unregisteredRenting
+        withOthers
+        livesWithParents
+        unknown
+        type
+      }
+      cohabitationAid {
+        ownPlace
+        registeredRenting
+        unregisteredRenting
+        withOthers
+        livesWithParents
+        unknown
+        type
+      }
+    }
+  }
+`
+
+export const useMunicipalities = () => {
+  const storageKey = 'currentMunicipalities'
```

Comment:
SHould this be in constants?

Reply:
Its not really used anywhere else, so I am not sure if that is necessary? 🙃